.github/workflows/coverity.yml

```diff
@@ -8,10 +8,11 @@ permissions: read-all
 
 jobs:
   coverity:
+    if: github.repository == 'intel/llvm'
     name: Coverity
     runs-on: [Linux, build]
     container:
-      image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:alldeps
+      image: ghcr.io/intel/llvm/ubuntu2404_build:latest
```

Comment:
Why this? We want to gradually switch to "alldeps" everywhere.

Reply:
`ubuntu2404_intel_drivers` is the only one that have `alldeps`, right?

Reply:
Moved back to alldeps - addd00f